Index: local.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>## This file must *NOT* be checked into Version Control Systems,\r\n# as it contains information specific to your local configuration.\r\n#\r\n# Location of the SDK. This is only used by Gradle.\r\n# For customization when using a Version Control System, please read the\r\n# header note.\r\n#Tue Jun 04 16:22:37 SGT 2024\r\nsdk.dir=C\\:\\\\Users\\\\liang\\\\AppData\\\\Local\\\\Android\\\\Sdk\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/local.properties b/local.properties
--- a/local.properties	(revision 2275dc838b9c442cd43f7c46d58180079997edf9)
+++ b/local.properties	(date 1719065560445)
@@ -4,5 +4,5 @@
 # Location of the SDK. This is only used by Gradle.
 # For customization when using a Version Control System, please read the
 # header note.
-#Tue Jun 04 16:22:37 SGT 2024
-sdk.dir=C\:\\Users\\liang\\AppData\\Local\\Android\\Sdk
+#Sat Jun 22 22:12:40 SGT 2024
+sdk.dir=C\:\\Users\\isaac\\AppData\\Local\\Android\\Sdk
